pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel_test.go

```diff
@@ -283,6 +373,7 @@ func TestAllCases(t *testing.T) {
 		r := &Reconciler{
 			multiChannelMessageHandler: NewFakeMultiChannelHandler(),
 			eventDispatcherConfigStore: channel.NewEventDispatcherConfigStore(logger),
+			messagingClientSet:         eventingclient.Get(ctx).MessagingV1(),
```

Comment:
@vaikas I'm still not certain I understand why `eventingclient` is used instead of `fakeclient` here (see previous comments).  It's probably me being too slow, but would you mind clarifying?